### PR TITLE
common.xml: Add a status flag for generic motor malfunction

### PR DESCRIPTION
**Pull request by amilcarlucas**
GCS and/or companion computers can then warn the user or take automated action

### CODE REVIEW

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -211,6 +211,9 @@
       <entry value="536870912" name="MAV_SYS_STATUS_OBSTACLE_AVOIDANCE">
         <description>0x20000000 Avoidance/collision prevention</description>
       </entry>
+      <entry value="1073741824" name="MAV_SYS_STATUS_SENSOR_MOTOR_MALFUNCTION">
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml:214]: @amilcarlucas Should this be `MAV_SYS_STATUS_MOTOR_MALFUNCTION` (i.e. why "Sensor"?)

@julianoes @auturgy This sounds useful. Any objection?

Comment (julianoes) [reply message_definitions/v1.0/common.xml:214]: No objections.

**Comment by amilcarlucas** [comment]
done

**Comment by amilcarlucas** [comment]
Checks passed and approved, can this be merged now?

File: message_definitions/v1.0/common.xml [hunk 2]
```diff
@@ -211,6 +211,9 @@
       <entry value="536870912" name="MAV_SYS_STATUS_OBSTACLE_AVOIDANCE">
         <description>0x20000000 Avoidance/collision prevention</description>
       </entry>
+      <entry value="1073741824" name="MAV_SYS_STATUS_MOTOR_MALFUNCTION">
+        <description>0x40000000 motor malfunction detected</description>
+      </entry>
```
Comment (Jaeyoung-Lim) [inline message_definitions/v1.0/common.xml:216]: Is this a general actuator failure or "just" for motors?

Comment (amilcarlucas) [reply message_definitions/v1.0/common.xml:216]: general porpulsion failure

Comment (Jaeyoung-Lim) [reply message_definitions/v1.0/common.xml:216]: In that case isn't `MOTOR` a little misleading?

Comment (amilcarlucas) [reply message_definitions/v1.0/common.xml:216]: Thanks for the feedback. Corrected and improved the comit message

File: message_definitions/v1.0/common.xml (hunk 1 above)
Comment (amilcarlucas) [reply message_definitions/v1.0/common.xml]: It is sensor because that is the name of the enum, and that is a naming convension in mavlink

**Comment by amilcarlucas** [comment]
ping, I would like to port this downstream at some time.

**Review by julianoes — APPROVED** [review]

**Pull request merged**